test/runtests.jl

```diff
@@ -1,6 +1,15 @@
+using Bijectors
+using Compat: only
+using Distributions
+using Optim
 using ParameterHandling
 using Test
+using Zygote
+
+using ParameterHandling: value
+using ParameterHandling.TestUtils: test_flatten_interface, test_parameter_interface
 
 @testset "ParameterHandling.jl" begin
-    # Write your own tests here.
+    include("flatten.jl")
```

Comment:
Is there call for a separate set of tests for `unflatten`?

Reply:
I don't think so (?) -- there are already `unflatten` tests wrapped up in the `flatten` tests, since `flatten` outputs `unflatten`.

Reply:
Looking again at the interface tests for flatten, I agree those should be sufficient.